jaxrs/pom.xml

```diff
@@ -13,6 +13,11 @@
 
     <artifactId>problem-details-jaxrs</artifactId>
     <description>An implementation for JAX-RS using an ExceptionMapper</description>
+    
+    <properties>
```

Comment:
Shouldn't the properties defined in the parent pom be adjusted instead?

Reply:
That's ok for me, but then even the spring version would be impacted.

Reply:
But both properties are not used by the spring modules